add-on/preferences/prefs.js

```diff
@@ -34,3 +33,6 @@ pref("loop.facebook.enabled", true);
 #else
 pref("loop.facebook.enabled", false);
 #endif
+pref("loop.facebook.appId", "1519239075036718");
+pref("loop.facebook.shareUrl", "https://www.facebook.com/dialog/send?app_id=%APP_ID%&link=%ROOM_URL%&redirect_uri=%REDIRECT_URI%");
```

Comment:
I believe it's better to keep it separated. It's easier to read, and to modify it if needed. 
Parameter replacement feel cleaner too.